egs/iam/v1/local/gen_topo.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
```

Comment:
Is this script used anywhere?

Reply:
yeah, it is used in local/prepare_lang. It is used for getting smaller hmm topology for punctuations.

Reply:
Oh I see. The right way to do this is to use the standard prepare_lang.sh (in utils) and then call gen_topo.py separately to write the topo in the lang directory. (Note: the topo does not affect any other file in the lang directory, it's only used later when initializing a gmm model). Please also remove local/prepare_lang.sh.

Reply:
ok, thank you. I will do it this way.